arcade/arcade/core/catalog.py

```diff
@@ -146,6 +176,9 @@ def add_tool(
         if fully_qualified_name in self._tools:
             raise KeyError(f"Tool '{definition.name}' already exists in the catalog.")
 
+        if str(fully_qualified_name).lower() in self._disabled_tools:
+            return
```

Comment:
Should we add a log here that says something like "Tool Toolkit.Name is disabled" to avoid confusion?

Reply:
Added a log, thanks

```
INFO  [15:29:38.332] Tool 'Math.Add' is disabled and will not be cataloged.
INFO  [15:29:38.332] Tool 'Math.Subtract' is disabled and will not be cataloged.
INFO  [15:29:38.333] Tool 'Math.Divide' is disabled and will not be cataloged.
INFO  [15:29:38.334] Tool 'Math.Sqrt' is disabled and will not be cataloged.
INFO  [15:29:38.535] Tool 'Spotify.GetAvailableDevices' is disabled and will not be cataloged.
INFO  [15:29:38.536] Serving the following toolkits:
INFO  [15:29:38.536]   - slack (arcade_slack): 2 tools
INFO  [15:29:38.536]   - math (arcade_math): 5 tools
INFO  [15:29:38.536]   - spotify (arcade_spotify): 12 tools
```